src/content/admin/registry-credentials/index.mdx

```diff
@@ -88,25 +88,32 @@ If you add credentials using CRDs they will be displayed in the UI, but they can
 Okteto runs a dedicated Kubernetes Controller to manage Registy Credentials. As part of this process, the Controller creates and manages a [Docker Config JSON](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets) secret in the Okteto namespace.
 This secret is called `okteto-dockerconfig-static` and is always up to date with your Registy Credentials, either through the UI or using CRDs.
 
-This docker config secret is used as a pull secret in installer jobs.
-If the `daemonset.configurePrivateRegistriesInNodes.enabled` setting is active, Okteto writes this secret on every node which allows Kubelet to pull private images at deployment time.
+This docker config secret is used as a pull secret in installer jobs, in case your installer job image is private.
+
+The docker config secret is also replicated in all namespaces and preview environments created by Okteto, with the name `okteto-regcred`.
```

Comment:
I think "replicated" is misleading because the contents are merged with the okteto registry (as described in the note below)